Asterisk/run.sh

```diff
@@ -51,10 +51,20 @@ bashio::var.json \
     -template /usr/share/tempio/http.conf.gtpl \
     -out /etc/asterisk/http.conf
 
+# Make a loop for these
+
 tempio \
     -template /usr/share/tempio/rtp.conf.gtpl \
     -out /etc/asterisk/rtp.conf
 
+tempio \
+    -template /usr/share/tempio/sip_custom.conf.gtpl \
+    -out /etc/asterisk/sip_custom.conf
```

Comment:
Unless you plan to add templating capabilities, I think we don't need tempio for such custom files.

Reply:
true, we have more files like that. Maybe we should move them to rootfs/etc/asterisk?

Reply:
Moved to `/etc/asterisk` in https://github.com/TECH7Fox/Asterisk-add-on/pull/27/commits/2900e08f9f4ee83bc4694b16c07ed6c9a04c4793